.github/workflows/build-ios-release-pullrequest.yml

```
@@ -18,14 +18,42 @@ jobs:
+          if [ -n "${GITHUB_HEAD_REF}" ]; then
+            git fetch origin ${GITHUB_HEAD_REF}:${GITHUB_HEAD_REF}
+            git checkout ${GITHUB_HEAD_REF}
+          else
+            git fetch origin ${GITHUB_REF##*/}:${GITHUB_REF##*/}
+            git checkout ${GITHUB_REF##*/}
+          fi
+          echo "Checked out branch: $(git rev-parse --abbrev-ref HEAD)"
+
+      - name: Get Latest Commit Details
+        id: get_latest_commit_details
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:3:44: Double quote to prevent globbing and word splitting

(shellcheck)